ivy_tests/test_ivy/test_functional/test_core/test_elementwise.py

```diff
@@ -971,19 +971,30 @@ def test_isfinite(
     )
 
 
+@st.composite
+def _ishelp(draw):
+    dtype, x, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"), ret_shape=True
+        )
+    )
+    dtype2, where = draw(
```

Comment:
I think the test should be made more exhaustive by generating `where` argument of all broadcastable shapes from `shape`. We can use helpers like [this](https://github.com/unifyai/ivy/blob/5d237bccc2ca36927b3f170c81aae331a7406d47/ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py#L1698) or [this](https://github.com/unifyai/ivy/blob/5d237bccc2ca36927b3f170c81aae331a7406d47/ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py#L1670) to generate such shapes